adding precisions (#5454)
DataDog/documentation: content/en/synthetics/private_locations.md

## Setup

> The private location worker is shipped as a Docker container, so you can run in on Linux based OS or Widows OS as long as you have the docker engine available on your host.

Author: ```suggestion
The private location worker is shipped as a Docker container, so you can run in on Linux based OS or Windows OS as long as you have the docker engine available on your host.
```

> The private location worker is shipped as a Docker container, so you can run in on Linux based OS or Windows OS as long as you have the docker engine available on your host.

Reviewer: ```suggestion
The private location worker is shipped as a Docker container, so it can run on a Linux based OS or Windows OS if the Docker engine is available on your host.
```

    docker run --init --rm -v $PWD/worker-config-<LOCATION_ID>.json:/etc/datadog/synthetics-check-runner.json datadog/synthetics-private-location-worker
    ```

>     **Note**: To scale a private location, add or remove workers on your host. It is possible to add several workers for one private location with one single configuration file. Each worker would then requests X requests depending on its number of free slots and when worker 1 is processing tests, worker 2 will request the following tests, etc..

Reviewer: ```suggestion
    **Note**: To scale a private location, add or remove workers on your host. It is possible to add several workers for one private location with one single configuration file. Each worker would then request `N` requests depending on its number of free slots and when worker 1 is processing tests, worker 2 requests the following tests, etc.
```
